core/controllers/feedback.py

```diff
@@ -146,6 +146,17 @@ def post(self, exploration_id):
         self.render_json(self.values)
 
 
+class ReportHandler(base.BaseHandler):
+    """"Handles operations relating to learner reports."""
+
+    @base.require_user
```

Comment:
I was wondering if we can give the ability to flag to only registered users and avoid probable spam attacks?